packages/terra-form-fieldset/src/Fieldset.jsx

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable react/jsx-boolean-value, jsx-a11y/label-has-for */
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+
+import 'terra-base/lib/baseStyles';
+import styles from './Fieldset.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Children the Field contains.
+   */
+  children: PropTypes.node,
+  /**
+   * Help element to display with the input.
+   */
+  help: PropTypes.node,
+  /**
+   * Determines whether the fieldset is an inline fieldset.
+   */
+  isInline: PropTypes.bool,
+  /**
+   * Legend for the input group.
+   */
+  legend: PropTypes.string,
+  /**
+   * Attributes to attach to the legend.
+   */
+  legendAttrs: PropTypes.object,
+  /**
+   * Determines whether the fieldset is required.
+   */
+  required: PropTypes.bool,
+};
+
+const defaultProps = {
+  children: null,
```

Comment:
lets remove the defaults for children & help & legend

Reply:
You can remove them for now. 

Given our talk in the war room on Friday and with other architects on the team, there is a chance we may introduce the old AirBnb lint rule for making sure defaults are declared.